sql/codegen.go

```diff
@@ -68,7 +68,7 @@ type filler struct {
 	Y                  *featureMeta
 	TableName          string
 	modelConfig
-	connectionConfig
+	*connectionConfig
```

Comment:
Why changing to a pointer?

Reply:
To improve reusability, `fillDatabaseInfo()` is renamed to `newConnectionConfig()` which returns a pointer.